### ISSUE TITLE
Why the model on the test performence soooo bad?

### ISSUE THREAD

**Issue by XinlongCui** [juntaoJianggavin/RWKV-UNet#4]
Here is the training code
python train.py --dataset Synapse --max_epochs 60 --base_lr 0.001 --img_size 224 --pretrained_path Pretrained_weights_encoder/net_B.pth

[03:31:28.147] Namespace(Dataset=<class 'datasets.dataset_synapse.Synapse_dataset'>, base_lr=0.001, batch_size=24, dataset='Synapse', deterministic=1, exp='rwkvSynapse224', img_size=224, is_pretrain=True, is_savenii=False, list_dir='./lists/lists_Synapse', max_epochs=60, max_iterations=20000, num_classes=9, path_specific='/home/cxl/RWKV-Unet/outputs/rwkvSynapse224/exp1_pretrain_epo60_bs24_lr0.001_224/epoch_49.pth', seed=1234, test_save_dir='../predictions', volume_path='./data/Synapse/test_vol_h5', z_spacing=1)
[03:31:28.147] exp1_pretrain_epo60_bs24_lr0.001_224
[03:31:28.148] 12 test iterations per epoch
[03:31:54.765] idx 0 case case0008 mean_dice 0.000345 mean_hd95 39.161724
[03:32:07.456] idx 1 case case0022 mean_dice 0.000329 mean_hd95 35.600650
[03:32:22.327] idx 2 case case0038 mean_dice 0.000274 mean_hd95 35.632707
[03:32:51.150] idx 3 case case0036 mean_dice 0.000149 mean_hd95 38.764715
[03:33:13.684] idx 4 case case0032 mean_dice 0.000355 mean_hd95 35.104531
[03:33:34.588] idx 5 case case0002 mean_dice 0.000268 mean_hd95 37.434109
[03:33:49.405] idx 6 case case0029 mean_dice 0.000422 mean_hd95 34.395221
[03:34:19.550] idx 7 case case0003 mean_dice 0.000170 mean_hd95 37.432230
[03:34:41.901] idx 8 case case0001 mean_dice 0.000545 mean_hd95 36.182912
[03:35:03.589] idx 9 case case0004 mean_dice 0.000425 mean_hd95 36.332632
[03:35:15.802] idx 10 case case0025 mean_dice 0.000264 mean_hd95 35.941358
[03:35:29.346] idx 11 case case0035 mean_dice 0.000204 mean_hd95 37.293389
[03:35:29.379] Mean class 1 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Mean class 2 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Mean class 3 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Mean class 4 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Mean class 5 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Mean class 6 mean_dice 0.002499 mean_hd95 292.850787
[03:35:29.379] Mean class 7 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Mean class 8 mean_dice 0.000000 mean_hd95 0.000000
[03:35:29.379] Testing performance in best val model: mean_dice : 0.000312 mean_hd95 : 36.606348

**Comment by juntaoJianggavin** [comment]
Can you provide the training logs?

**Comment by beaconwolfs** [comment]
我也遇到了这个问题 请问你解决了吗  然后他是因为什么原因 导致的
